manifests/kustomize/base/cache-deployer/cluster-scoped/cache-deployer-clusterrole.yaml

```diff
@@ -15,10 +15,20 @@ rules:
   - delete
   - get
   - update
+  - list
```

Comment:
Why was list and watch needed?
They were not mentioned in your PR's description error message.

Reply:
It comes from https://kubernetes.io/docs/reference/access-authn-authz/certificate-signing-requests/#authorization

Reply:
Can you confirm if this works without them?
as far as I know, current permissions are enough, we are not listing or watching them

Reply:
I confirm. Updated PR